src/main/java/com/jland/repository/mapper/PresentationMapper.java

```diff
@@ -0,0 +1,45 @@
+package com.jland.repository.mapper;
+
+import com.jland.model.Event;
+import com.jland.model.Presentation;
+import com.jland.model.User;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalTime;
+
+public class PresentationMapper implements RowMapper<Presentation> {
+
+    @Override
+    public Presentation mapRow(ResultSet resultSet, int i) throws SQLException {
+        Presentation presentation = new Presentation();
+        presentation.setId(resultSet.getLong("presentation_id"));
+        presentation.setName(resultSet.getString("presentation_name"));
+        Event event = getEvent(resultSet);
+        presentation.setEvent(event);
+        User user = getUser(resultSet);
+        presentation.setUser(user);
+        return presentation;
+    }
+
+    private Event getEvent(ResultSet resultSet) throws SQLException {
```

Comment:
I think to make sense have separate EventMapper as far as we have event table in our database

Reply:
For now we don't use ORM and it's unnecessary to split mapper for several classes. When we migrate to spring data, we will refactor this part